.taskcluster.yml

```diff
@@ -84,7 +84,7 @@ tasks:
                       $if: 'tasks_for == "cron"'
                       then: '${ownTaskId}'
           in:
-              $if: 'tasks_for in ["github-pull-request", "github-release", "action", "cron"] || tasks_for == "github-push" && head_branch[:10] != "refs/tags/"'
+              $if: 'tasks_for in ["github-pull-request", "action", "cron"] || (tasks_for == "github-push" && head_branch[:10] != "refs/tags/") || (tasks_for == "github-release" && event["action"] == "published")'
```

Comment:
Bug: This line will bust the json-e parser in the case of `cron`. I hit this very problem, yesterday. I ended up fixing it this way https://github.com/mozilla-mobile/android-components/pull/4591/files#diff-ac0229d1171c0983b27003af4c7441a5R98. Tell me what you think!

Reply:
Good call! I always think that [`json-e` handles short-circuit evaluation](https://github.com/taskcluster/json-e/issues/244)